AABB_tree/include/CGAL/AABB_tree.h

```diff
@@ -484,15 +475,23 @@ namespace CGAL {
     }
 
     // clears internal KD tree
-    void clear_search_tree() const
+    void clear_search_tree()
     {
+#ifdef CGAL_HAS_THREADS
+      if ( m_atomic_search_tree_constructed.load() )
```

Comment:
As you choose to use the full API of `std::atomic`, you can as well pass the right `std::memory_order` argument:
```suggestion
      if ( m_atomic_search_tree_constructed.load(std::memory_order_acquire) )
```

Reply:
I think it should be relaxed since the variable is guaranteed not to be modified once we are in this function (which I thought was the default but is seq actually...).

Reply:
That is right (relaxed). I had not seen in which function we were. Actually, in this function, the synchronization between threads and the performance do not really matter.